source/simulate/run-simulation.md

```diff
@@ -150,310 +149,208 @@ Let's also say you have the following `data.csv` and that you want to calculate
-- or provide inputs in bulk, typically using tabular data (CSV, Excel, etc.)
-
-A third possibility exists: you can set up a small-scale situation, as in test cases; and use it to generate a number of "copies" of this situation, in which one or more variables of your choice take on a range of values.
-
-We do this by adding an "axes" entry to a test case:
-
+And, persons' order is kept:
 ```python
-WITH_AXES = {
-    'persons': {'Ari': {}, 'Paul': {}, 'Leila': {}, 'Javier': {}},
-    'households': {
-        'h1': {'children': ['Leila'], 'parents': ['Ari', 'Paul']},
-        'h2': {'parents': ['Javier']}
-        },
-    'axes': [[{'count':10, 'name':'salary', 'min':0, 'max':3000, 'period':'2018-11'}]]
-    }
+>>> print(data.person_id.values)
 
-simulation_builder = SimulationBuilder()
-simulation = simulation_builder.build_from_entities(tax_benefit_system, WITH_AXES)
-```
-
-Be careful to note the structure of the "axes" field: an **array of arrays** of axis objects. We will come back to this, but for now let's investigate the effects of adding this single axis.
-
-As before, `BASIC_TEST_CASE` describes one household with two parents and one child, and a second household which is in fact a single adult person. However, the specification of an axis results in something quite different now:
+array([ 1  2  3  4  5  6  7 12  8  9 10 11])
+``` 
 
+Thus, you can get the calculated `income_tax` of one person. For example, get its value for the 8th person in the list with:
 ```python
->>> len(simulation.calculate('salary', '2018-11'))
-40
-```
-
-We started with a "prototype" situation containing 4 individuals, and we specified an axis which replicates this situation 10 times. And so, as expected, we end up with a simulation containing 4 times 10 individuals. 
+>>> print(income_tax.item(7))  # person_id == 12
 
-What happened with respect to the data? It's easier to represent if we first "reshape" the computed data to reflect this structure of 10 groups of 4 individuals:
-
-```python
->>> numpy.reshape(simulation.calculate('salary', '2018-11'),(10,4))
-array([[   0.     ,    0.     ,    0.     ,    0.     ],
-       [ 333.33334,    0.     ,    0.     ,    0.     ],
-       [ 666.6667 ,    0.     ,    0.     ,    0.     ],
-       [1000.     ,    0.     ,    0.     ,    0.     ],
-       [1333.3334 ,    0.     ,    0.     ,    0.     ],
-       [1666.6666 ,    0.     ,    0.     ,    0.     ],
-       [2000.     ,    0.     ,    0.     ,    0.     ],
-       [2333.3333 ,    0.     ,    0.     ,    0.     ],
-       [2666.6667 ,    0.     ,    0.     ,    0.     ],
-       [3000.     ,    0.     ,    0.     ,    0.     ]], dtype=float32)
+180.0
 ```
 
-We can see that, for the requested period, the variable `salary` **of the first individual in each group** is varying in increments from 0 to 3000.
-
-### Targeting individuals
+#### Application with multiple entities: calculate a population's households total taxes from a CSV file
```

Comment:
Same comment as before, it is not very clear and the phrase is a bit lengthy.

Reply:
Restored to previous title.